sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -124,15 +122,16 @@ export class QueueClient implements Client {
    *
    * @returns Receiver A receiver to receive messages from a Queue which does not have
    * sessions enabled.
+   * @throws {Error} Thrown if the QueueClient or the underlying connection is closed.
+   * @throws {Error} Thrown if an open receiver already exists on the QueueClient.
+   * @throws {MessagingError} Throws this error with name `InvalidOperationError` if the Queue has sessions enabled
```

Comment:
@bterlson, Any wordsmithing advice here where we have a generic `MessagingError` with different names?

Reply:
Here's what I'd like to see, though alignment on tense and removal of `thrown if`/`throws this error if` can come later if you like.

Basic idea: add a `when` clause to the beginning.

```
* @throws {Error} The QueueClient or the underlying connection is closed.
* @throws {Error} An open receiver already exists on the QueueClient.
* @throws {MessagingError} When the error's `name` property is `InvalidOperationError`, the Queue has sessions enabled and no `sessionOptions` were provided.
```

Reply:
Is this to update all of the error messages to use this convention?
Would apply to EventHubs as well, if so should we do it in this PR? (can do it once other comments are resolved)

Reply:
The change from using "Thrown" to not using thrown can be done in a separate PR.
Let's first see how the current way of rendering `@throws` shows up in docs.microsoft.com
We might want to do tweaks based on that which can be combined with the above wording changes.

Reply:
Recent chat seems like we are inclined towards removing 'Thrown' and braces? Or do want to keep the improvement to across SDKs later?

Reply:
Improvement across SDKs will be taken up later. Current format in this PR is good enough for now.